app/models/user.rb

```diff
@@ -130,6 +125,18 @@ def name_initials
     full_name.split.map { |part| part[0] }.join.upcase
   end
 
+  def deactivate
```

Comment:
Oh these are your methods :D
Can we use ! on these given they are writing to the db?

Reply:
See other comments. I'll update these to `update!` too

Reply:
81c23a7430a2 adds bangs to `update!`